test/common/http/conn_manager_utility_test.cc

```diff
@@ -167,32 +172,32 @@ TEST_F(ConnectionManagerUtilityTest, EdgeRequestRegenerateRequestIdAndWipeDownst
   {
     TestHeaderMapImpl headers{{"x-envoy-downstream-service-cluster", "foo"},
                               {"x-request-id", "will_be_regenerated"}};
-    EXPECT_CALL(random_, uuid()).WillOnce(Return(generated_uuid));
+    EXPECT_CALL(random_, uuid()).Times(1);
```

Comment:
Is it, though? I was under impression that `Times(1)` requires `random_.uuid()` to be called **exactly once** whereas without this, it can be called multiple times?